src/containers/consignment.rs

```diff
@@ -389,14 +389,14 @@ impl<const TRANSFER: bool> Consignment<TRANSFER> {
         }
     }
 
-    /// Method to construct valid consignment which can be consumed by a stash
-    /// for debug purposes.
+    /// Method to forcefully construct a valid consignment, which can be consumed by a stash.
     ///
```

Comment:
Sorry, I strictly disagree with this. It has - and must not have (!) no other use than debug purposes, since any other use - is security leak. Literally, my opinion is so strong, that until the public audit conclusion on this issue I will just fork this repository and will contribute to the fork not having this security issue.